src/app/file-manager/services/register-download.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { ElectronService } from 'ngx-electron';
+import { SecurityService } from './security.service';
+import { Item } from '../models/item';
+
+@Injectable()
+export class RegisterDownloadService {
+
+  constructor(private electronService: ElectronService) { }
+
+  public _downloadEvent(items: Item[]) {
```

Comment:
Why is this called `_downloadEvent()` instead of `downloadEvent()`? Does it conflict with another name? Also it seems like a better name for this is `startDownload()`

Reply:
No, it's just a convention in some libraries that I follow. I'll change it.